### ISSUE TITLE
Copter: Specify SMART RTL in flight plan

### ISSUE THREAD

**Issue by muramura** [ArduPilot/ardupilot#21204]
## Feature request

**Is your feature request related to a problem? Please describe.**

I have selected RETURN_TO_LAUNCH in my flight plan.
I have returned through airspace I have not flown through.

**Describe the solution you'd like**

I want to use SMART RTL in my flight plan.

**Describe alternatives you've considered**

None

**Platform**
[  ] All
[  ] AntennaTracker
[X] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

**Additional context**

![Screenshot from 2022-07-17 07-18-37](https://user-images.githubusercontent.com/646194/179388477-d8b2d356-49cb-47e3-bd6a-8becfa2345ba.png)

**Comment by rmackay9** [comment]
Yes, I agree that SMART_RTL should be added as a navigation command so it can be added to missions independently of RTL.
